packages/excalidraw/types.ts

```diff
@@ -284,6 +284,7 @@ export interface AppState {
    */
   editingTextElement: NonDeletedExcalidrawElement | null;
   editingLinearElement: LinearElementEditor | null;
+  elbowLinearElement: LinearElementEditor | null;
```

Comment:
this is a bit confusing (mainly because I don't quite understand it fully yet). AFAIK it's only being set when dragging elbow points that aren't start/end points.

We should definitely distinguish it from `selectedLinearElement`, and we also should not allow potentially setting (or forgetting to unset) both `elbowLinearElement` & `editingLinearElement`.

Why not reuse `editingLinearElement` for this and typecheck?

Reply:
This is the state race condition or similar issue I mentioned. I don't know how/when/where it goes haywire, but at some cases it contains snapshots of midpoint positions at random from the previous render frames. I can certainly debug it and then we can use `selectedLinearElement`. I wouldn't use `editingLinearElement`, that is for the edit mode for linear elements, which elbow arrows don't support.